docs/fleet/machines.md

```diff
@@ -92,6 +92,11 @@ For local communication between [parts](/build/configure/parts-and-remotes/#mach
 <br>
 
 You can also access the control interface using the [Viam mobile app](/fleet/#the-viam-mobile-app), which you can find on the [App Store](https://apps.apple.com/vn/app/viam-robotics/id6451424162) and on [Google Play](https://play.google.com/store/apps/details?id=com.viam.viammobile&hl=en&gl=US).
+The remote control on the mobile app is organized by location and organization, simplifying the process of managing your fleet of machines.
+Extend this control to your team by granting them access to your organization.
+Additionally, you can search for machines in your fleet, as well as securely upload images from your phone to the Cloud, which you can view on the Viam app's [**DATA** tab](https://app.viam.com/data/) for later use.
```

Comment:
I like that you've worded this in a very action-oriented way, however I think this can be paired down since it's sort of just mirroring what app.viam.com does anyway as far as organizations and locations. The first sentence already says the mobile app allows you to access the control interface, so I *think* readers can assume that it's a similar experience, i.e. that they can access the control interface for any machine they could access in app.viam.com, with the same permissions. I don't think we need to highlight data upload here since this section is specifically about the control tab, and we talk about data upload here https://docs.viam.com/data/upload/#upload-images-with-the-viam-mobile-app. One possibility:
```suggestion
You can also control any of the machines you have access to using the [Viam mobile app](/fleet/#the-viam-mobile-app), which you can find on the [App Store](https://apps.apple.com/vn/app/viam-robotics/id6451424162) and on [Google Play](https://play.google.com/store/apps/details?id=com.viam.viammobile&hl=en&gl=US).
```
Or perhaps:
```suggestion
You can also access the control interface using the [Viam mobile app](/fleet/#the-viam-mobile-app), which you can find on the [App Store](https://apps.apple.com/vn/app/viam-robotics/id6451424162) and on [Google Play](https://play.google.com/store/apps/details?id=com.viam.viammobile&hl=en&gl=US).
The mobile app is a convenient way to control your machines when you're not at your desk.
```